docker/dockerfile_release.template

```diff
@@ -0,0 +1,47 @@
+FROM docker.io/pytorch/pytorch:1.10.0-cuda11.3-cudnn8-runtime
```

Comment:
Added https://github.com/pytorch/pytorch/issues/67966 requesting 1.9.1

Reply:
PyTorch 1.9.1 images are pushed, although with CUDA 11.1. If it is possible to just update CUDA, that would speed up builds I believe, because `pip` is handling PyTorch downgrade very slowly.